src/components/Articles.jsx

```diff
@@ -1,9 +1,53 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
+import Article from "./article";
 
 export default function Articles() {
+  const [articles, setArticles] = useState([]);
+  useEffect(() => {
+    fetch("https://news-3fmi.onrender.com/api/articles")
+      .then((response) => response.json())
```

Comment:
I'd separate your API calls into a separate file with a base URL to make your code more manageable.
Have a look at these notes: https://notes.northcoders.com/courses/js-front-end/react-router

Let me know if you need any help with this 😄